### PR TITLE
Fix examples further to CodeMirror API change

### PR DESCRIPTION
**Pull request by echarles**
## References

Fixes https://github.com/jupyterlab/jupyterlab/issues/14221

## Code changes

Update the cell and notebook examples

## User-facing changes

The outputs are now displayed as before for the `cell`, `notebook` and `console` examples..

Untitled6

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @echarles for looking into it :+1: 

Looks like it needs a lint check, but looking good otherwise.

**Comment by echarles** [comment]
Thx @jtpio. Just linted and pushed.

File: packages/codemirror-extension/src/services.tsx [hunk 1]
```diff
@@ -200,15 +200,15 @@ export const extensionPlugin: JupyterFrontEndPlugin<IEditorExtensionRegistry> =
   };
 
 /**
- * CodeMirror real-time collaboration binding provider.
+ * CodeMirror binding provider.
```
Comment (fcollonval) [inline packages/codemirror-extension/src/services.tsx:203]: Could we keep a reference to what it is bound to CodeMirror?

```suggestion
 * CodeMirror shared model binding provider.
```

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @echarles 

I have one small comments and the releaser test needs to be fixed - the explicit dependency to `@jupyter/ydoc` should be provided in the examples package.json.

> the other test failures are due to flaky errors or are not the consequence of this particular PR.

**Comment by echarles** [comment]
> the explicit dependency to @jupyter/ydoc should be provided in the examples package.json.

thx for the review @fcollonval - I have added the needed deps.

**Comment by fcollonval** [comment]
Almost there @echarles, you need to run `jlpm install` to update the yarn.lock file and this PR will be good to go.

> As you are submitting from an org, I unfortunately cannot help without those changes.

**Comment by echarles** [comment]
@fcollonval Thx for your help and patience :)

**Review by fcollonval — APPROVED** [review]
Thanks @echarles 

> CI failures are not related

**Pull request merged**